alicloud/table_alicloud_rds_instance.go

```diff
@@ -440,33 +441,12 @@ func tableAlicloudRdsInstance(ctx context.Context) *plugin.Table {
 				Description: "The availability status of the instance. Unit: %.",
 			},
 			{
-				Name:        "db_instance_ip_array_name",
-				Type:        proto.ColumnType_STRING,
-				Hydrate:     getRdsInstanceIPArrayList,
-				Transform:   transform.FromField("DBInstanceIPArrayName"),
-				Description: "The name of the IP address whitelist.",
-			},
-			{
-				Name:        "db_instance_ip_array_attribute",
-				Type:        proto.ColumnType_STRING,
-				Hydrate:     getRdsInstanceIPArrayList,
-				Transform:   transform.FromField("DBInstanceIPArrayAttribute"),
-				Description: "The attribute of the IP address whitelist.",
-			},
-			{
-				Name:        "security_ip_type",
-				Type:        proto.ColumnType_STRING,
+				Name:        "ip_array_list",
```

Comment:
Can you please rename this to `security_ips_src`? In this column, we should not modify any of the fields, even the `SecurityIPList` field, which is a comma delimited list of IPs.

Reply:
We should also add a column called `security_ips` that's a list of strings, which is composed of the addresses from the `SecurityIPList` fields.